resources/views/waitingrooms/partials/_form_split.blade.php

```diff
@@ -6,17 +6,17 @@
     <div class="container__block">
         <div class="form-item -padded">
             <label class="field-label" for="competition_end_date">End date for these competitions:</label>
-            <input type="date" name="competition_end_date" id="competition_end_date" value='MM/DD/YYYY' class="text-field"></input>
+            <input type="date" name="competition_end_date" id="competition_end_date" class="text-field" value="{{ format_date_form_field($room, 'competition_end_date') }}"></input>
         </div>
 
         <div class="form-item -padded">
             <label class="field-label" for="id">Max users per competition:</label>
-            <input type="number" name="competition_max" id="competition_max" class="text-field" value="300" data-total="{{count($room->users)}}" }/>
+            <input type="number" name="competition_max" id="competition_max" class="text-field" data-total="{{count($room->users)}}" placeholder=300 value="{{ old('competition_max') }}" />
```

Comment:
You want it to actually have a value of `300` in the input though, right? Then you should probs just not use the `placeholder` and do the other thing I mentioned with the `or` keyword. I just mentioned `placeholder` as an example. Placeholders are used when you want to provide an example to user of what the input should look like... like an email example placeholder, etc.